### ISSUE TITLE
Value Breakdown chart might be missing data

### ISSUE THREAD

**Issue by cameronmcefee** [TelemetryDeck/Viewer#358]
Insight ID: 6867C16F-4B40-4DFC-BADA-AEAC23BA79A6

Hello! I'm not sure if I've found a bug or I'm just misunderstanding how the Values Breakdown (Top N) table works when you specify a granularity. It switches to a bar chart. I'd expect every bar to be all the way to 100 but then further segmented by each value. However mine appear at different heights. At first I thought the empty gap might be "null" but null has a color in the key so I don't really understand what is going on.

![Screenshot 2024-02-26 at 10 05 58 AM](https://github.com/TelemetryDeck/Issues/assets/72919/144cf072-a32f-4fb5-9e4b-c8576974e48a)

**Comment by winsmith** [comment]
Yeah they should add up to 100%, so there seems to be something wrong. Investigating. 

Sidenote: the "null" is usually because you have a signal in the filtered data set that doesn't have a utm_campaign property, You can filter those out using a selector filter "utm_campaign=" and then negating that.
